packages/app-builder/src/services/editor/ast-editor.tsx

```diff
@@ -315,16 +317,56 @@ export function useRootOrAndChildValidation(stringPath: string) {
   ]);
 }
 
-export function useValidationStatus(stringPath: string) {
+export function useValidationStatus(
+  stringPath: string,
+  returnValue?: ReturnValue,
+) {
   const path = usePath(stringPath);
   const evaluationErrors = useEvaluationErrors(stringPath);
   const parentPath = useParentPath(path);
+  const parentNode = useParentAstNode(parentPath);
+  const isDivByZeroField =
+    isDivisionDenominator(parentNode, path) &&
+    !returnValue?.isOmitted &&
+    returnValue?.value === 0;
+
   const parentEvaluationErrors = useParentEvaluationErrors(parentPath);
+  const valueIsNull = !returnValue?.isOmitted && returnValue?.value === null;
   return React.useMemo(() => {
-    return getValidationStatus(
+    return getValidationStatus({
       evaluationErrors,
+      valueIsNull,
+      isDivByZeroField,
       parentEvaluationErrors,
-      parentPath?.childPathSegment,
-    );
-  }, [evaluationErrors, parentEvaluationErrors, parentPath]);
+      pathSegment: parentPath?.childPathSegment,
+    });
+  }, [
+    evaluationErrors,
+    isDivByZeroField,
+    parentEvaluationErrors,
+    parentPath?.childPathSegment,
+    valueIsNull,
+  ]);
+}
+
+function isDivisionDenominator(
+  parentNode: Tree<AstNode> | undefined,
+  path: Path,
+) {
+  if (path.length === 0) {
+    return false;
+  }
+  const pathLastPart = path[path.length - 1];
+  if (!pathLastPart) {
+    return false;
+  }
+  if (pathLastPart.type !== 'children') {
+    return false;
+  }
+
+  return (
+    parentNode?.name === '/' &&
```

Comment:
So, I tried to see if I could add a unique declaration of the known operators somewhere with aliased names, but I quickly realized that this would be too many places to modify code... possible refactoring better left for another PR.

Reply:
I don't see what you mean here. But this perfectly fine to me to use it like this here.

`/` is a value used in our serialization model, this won't change in the future, we do not need to create a variable to represent it IMO.

Reply:
It's fine and I'm not going to put too much effort into this right now, but it's a bit close to a "magic string" situation.
I wouldn't really create a _variable_ to represent it either, but a well named constant, may make sense.